### PR TITLE
updated mexican sound fx, simplified logic.

### PR DESCRIPTION
**Pull request by gubatron**
In comparison the .wav files might be a bit too heavy, I couldn't find a way with Audacity (which is the only sound manipulation I have here for .wav) to lower the quality/file size. If you tell me I can do that and make another pull request, or commit on this branch before merging it.

I've loaded all the sounds into an array and they're just played randomly, this also made sound loading and unloading simpler IMO.

Cheers

### CODE REVIEW

File: Project.xml [hunk 1]
```diff
@@ -192,6 +192,10 @@
 		<sound path="TrumpChina.wav" id="TrumpChina" />
 		<sound path="TrumpUghYaYe.wav" id="TrumpUghYaYe" />
 		<sound path="TrumpSitDown.wav" id="TrumpSitDown" />
+		<sound path="Laborer00.wav" id="Laborer00" />
+		<sound path="Laborer01.wav" id="Laborer01" />
+		<sound path="Laborer02.wav" id="Laborer02" />
+		<sound path="Laborer03.wav" id="Laborer03" />
 	</assets>
```
Comment (gubatron) [inline Project.xml:199]: I didn't see the older sounds in this file, therefore I couldn't remove them.

Comment (Werwolf696) [inline Project.xml:199]: Yeah, since they are not played with the global sound util (FlxG.sound.play), they aren't embedded.

**Comment by gubatron** [comment]
rebased the branch to keep it up to date with your master.

**Comment by gubatron** [comment]
wohoo, cheers! Thank you.

**Pull request merged**